BungeeCord-Patches/0064-Adventure.patch

```diff
@@ -148,7 +148,7 @@ index a4011335..e73f093d 100644
 +     * @param restricted whether the server will be restricted or not
 +     * @return the constructed instance
 +     */
-+    public abstract ServerInfo constructServerInfo(final @org.checkerframework.checker.nullness.qual.NonNull String name, final @org.checkerframework.checker.nullness.qual.NonNull InetSocketAddress address, final net.kyori.adventure.text.@org.checkerframework.checker.nullness.qual.NonNull Component motd, final boolean restricted);
++    public abstract @org.checkerframework.checker.nullness.qual.NonNull ServerInfo constructServerInfo(final @org.checkerframework.checker.nullness.qual.NonNull String name, final @org.checkerframework.checker.nullness.qual.NonNull InetSocketAddress address, final net.kyori.adventure.text.@org.checkerframework.checker.nullness.qual.NonNull Component motd, final boolean restricted);
```

Comment:
`final net.kyori.adventure.text.@org.checkerframework.checker.nullness.qual.NonNull Component`

I smell wrong search&replace

Reply:
If you're talking about the annotation going between the fully qualified package and the name, that's actually how you're meant to do it, and Java won't allow you to put the annotation at the front before you fully qualify the class name.

Reply:
wtf my bad